frontends/beta-frontend/shared/components/Grid/TeamColumn.tsx

```diff
@@ -24,12 +24,40 @@ export const teamColumn = [
   },
   { field: 'age', headerName: 'Age', headerAlign: 'center' as GridAlignment, align: 'center' as GridAlignment, minWidth: 70, flex: 1 },
   {
+    // field: 'position',
     field: 'position',
+    valueFormatter: (params: GridValueFormatterParams<any>) => {
+      switch (params.value) {
+        case 'GOALKEEPER':
+          return 'GK';
+        case 'DEFENDER':
+          return 'DF';
+        case 'MIDDLE':
+          return 'MD';
+        case 'FORWARD':
+          return 'FW';
+        default:
+          return params.value;
+      }
+    },
     headerName: 'Position',
     headerAlign: 'center' as GridAlignment,
     align: 'center' as GridAlignment,
     minWidth: 70,
     flex: 1,
+    // Assign Css class based on position
+    cellClassName: (params: GridCellParams<string>) => {
+      if (params.value == null) {
+        return '';
+      }
+
+      return clsx('super-app', {
+        goalkeeper: params.value === 'GOALKEEPER',
```

Comment:
when you use class names the same as a param.value you can just add `params.value.toLowerCase()` instead of adding multiple "the same" ifs

Reply:
Made the change on new commit